browser/brave_ads/ads_service_impl.cc

```diff
@@ -2098,6 +2105,87 @@ void AdsServiceImpl::RecordP2AEvent(const std::string& name,
   }
 }
 
+void AdsServiceImpl::LogTrainingCovariates(
+    const ads::mojom::TrainingCovariatesPtr training_covariates) {
+  if (!ad_notification_timing_data_store_) {
+    return;
+  }
+
+  // TODO(https://github.com/brave/brave-browser/issues/21189): Refactor DB to
+  // use generic key/value schema across all data stores
```

Comment:
Would be great if this tech debt could be picked up before adding new log entries in the future